kore/app/exec/Main.hs

```diff
@@ -698,13 +700,15 @@ koreProve execOptions proveOptions = do
     specModule <- loadModule specMainModule definition
     let KoreProveOptions{saveProofs} = proveOptions
     maybeAlreadyProvenModule <- loadProven definitionFileName saveProofs
+    let KoreProveOptions{maxCounterexamples} = proveOptions
```

Comment:
I don't remember, `kore-repl` uses a different type for its options, right? I'm just wondering if we'd run into the case where someone tries to run `kore-repl` with `--max-counterexamples`, and that doesn't really make sense.

Reply:
I moved it to `KoreExecOptions`. Do you think it's better this way?